src/utils/shortLink.ts

```diff
@@ -0,0 +1,89 @@
+import { ShortLinkPlatform as ShortLinkPlatformType } from '@graasp/sdk';
+
+import { BUILDER } from '@/langs/constants';
+
+// TODO: check where to store them
+export const MIN_SHORT_LINK_LENGTH = 6;
+export const MAX_SHORT_LINK_LENGTH = 255;
+
+export function randomString(length: number): string {
+  const characters =
+    'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+  let rand = '';
+
+  for (let i = 0; i < length; i += 1) {
+    const randomIndex = Math.floor(Math.random() * characters.length);
+    rand += characters.charAt(randomIndex);
+  }
+
+  return rand;
+}
+
+export function randomAlias(length: number = MIN_SHORT_LINK_LENGTH): string {
+  if (length < MIN_SHORT_LINK_LENGTH || length > MAX_SHORT_LINK_LENGTH)
+    throw new Error(
+      `Lenght must be between ${MIN_SHORT_LINK_LENGTH} and ${MAX_SHORT_LINK_LENGTH}`,
```

Comment:
```suggestion
      `Length must be between ${MIN_SHORT_LINK_LENGTH} and ${MAX_SHORT_LINK_LENGTH}`,
```
+ should we translate this?

Reply:
Normally, this error should never happen when generating new alias because the user can't modify input call. So I think it is not necessary to translate this error.